src/core/services/encryption/__tests__/encryption-init.provider.spec.ts

```
@@ -0,0 +1,71 @@
+    } as any;
+  });
+
+  it('should be defined as a provider with correct structure', () => {
+    expect(EncryptionInitProvider).toBeDefined();
+    expect(EncryptionInitProvider.provide).toBe('ENCRYPTION_INIT');
+    expect(EncryptionInitProvider.useFactory).toBeDefined();
+    expect(EncryptionInitProvider.inject).toEqual([EncryptionService]);
+  });
+
+  it('should call setEncryptionService with the encryption service instance', () => {
+    const factory = EncryptionInitProvider.useFactory as Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)